dask/array/creation.py

```diff
@@ -0,0 +1,134 @@
+from __future__ import absolute_import
+from itertools import count
+
+import numpy as np
+
+from .core import Array
+
+linspace_names = ('linspace-%d' % i for i in count(1))
+arange_names = ('arange-%d' % i for i in count(1))
+
+
+def _get_blocksizes(num, blocksize):
+    # compute blockdims
+    remainder = (num % blocksize,)
+    if remainder == (0,):
+        remainder = tuple()
+    blocksizes = ((blocksize,) * int(num // blocksize)) + remainder
+    return blocksizes
+
+
+def linspace(start, stop, blocksize, num=50):
```

Comment:
Is there a way to choose a default blocksize on the fly? Or do we just use some constant default?